policies_test.go

```diff
@@ -263,8 +263,12 @@ func TestSimpleRetryPolicy(t *testing.T) {
 		{5, false},
 	}
 
+	// initiate metrics map
```

Comment:
I think you can do away with this comment as well as the ones in lines 269 and 354.

Reply:
:-)